src/client/util/word.ts

```diff
@@ -39,8 +39,8 @@ export default function createWordSet(): Word[] {
 
 function buildWordSet(
   allWords: string[],
-  count: Number,
-  kind: WordKind
+  count: number,
```

Comment:
It was always wrong of me to use a `Number` here - we should use the `number` primitive instead.